physionet-django/project/forms.py

```diff
@@ -1041,3 +1047,52 @@ def save(self):
         invitation.added_date = timezone.now()
         invitation.save()
         return invitation
+
+
+class CustomClearableFileInput(forms.ClearableFileInput):
+    template_name = 'project/custom_clearable_file_input.html'
+    initial_text = 'Current file'
+    clear_checkbox_label = 'Remove file'
+
+
+class EthicsForm(forms.ModelForm):
+    class Meta:
+        model = ActiveProject
+        fields = ('ethics_statement',)
+        help_texts = {
+            'ethics_statement': (
+                '* A statement regarding ethical concerns for the work. '
+                'This statement will be published with the resource, '
+                'and typically describes formal approvals acquired for '
+                'the creation of the resource, and/or ethical considerations '
+                'for users of the resource. If no concerns, please indicate '
+                'this “The authors declare no ethics concerns.'
+            ),
+        }
+
+    def __init__(self, editable=True, **kwargs):
+        super().__init__(**kwargs)
+
+        if not editable:
+            for field in self.fields.values():
+                field.disabled = True
+
+
+class UploadedDocumentForm(forms.ModelForm):    
+    class Meta:
+        model = UploadedDocument
+        fields = ('document_type', 'document',)
+        widgets = {'document': CustomClearableFileInput}
+
+
+class UploadedDocumentFormSet(BaseGenericInlineFormSet):
+    form_name = 'project-uploadeddocument-content_type-object_id'
+    item_label = 'Supporting Documents'
+    max_forms = 3
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.help_text = (
+            'Upload supporting documents. For more information on '
+            'what documents are expected, see the [author guidance page].'
```

Comment:
can the [author guidance page] be changed to a hyperlink to the author guidance page? currently it's here: https://physionet.org/about/publish/#guidelines

also, add this sentence at the end: "These documents reviewed by the editorial staff, and are not published with the project."

Reply:
🆗